### PR TITLE
[1LP][RFR] Automate test case: test_change_truncate_long_text_save_button_enabled

### PR DESCRIPTION
**Pull request by valaparthvi**
This PR brings the following changes:
1. Automate test: `test_change_truncate_long_text_save_button_enabled`
2. Add `settings` tag to related test cases.

{{ pytest: cfme/tests/configure/test_visual_infra.py::test_change_truncate_long_text_save_button_enabled --use-template-cache -sqvvv }}

### CODE REVIEW

File: cfme/tests/configure/test_visual_infra.py [hunk 1]
```diff
@@ -365,8 +375,45 @@ def test_template_noquads(appliance, set_template_quad):
...
+        testSteps:
+            1. See if save button is enabled.
+            2. Save and check if the value was updated.
+        expectedResults:
+            1. Save button is enabled.
+            2. Value is changed successfully.
+    """
+    view = navigate_to(appliance.user.my_settings, "Visual")
+
+    available_options = [_.text for _ in view.tabs.visual.grid_tile_icons.long_text.all_options]
```
Comment (john-dupuy) [inline cfme/tests/configure/test_visual_infra.py]: **Optional** why not set a variable `visual = view.tabs.visual`?

File: cfme/tests/configure/test_visual_infra.py [hunk 2]
```diff
@@ -365,8 +375,45 @@ def test_template_noquads(appliance, set_template_quad):
...
         initialEstimate: 1/10h
+        tags: settings
     """
     view = navigate_to(appliance.collections.infra_templates, 'TemplatesOnly')
     view.toolbar.view_selector.select('Grid View')
     # Here data property will return an empty dict when the Quadrants option is deactivated.
     assert not view.entities.get_first_entity().data
+
+
+def test_change_truncate_long_text_save_button_enabled(appliance):
```
Comment (john-dupuy) [inline cfme/tests/configure/test_visual_infra.py]: **Optional**: maybe shorten this test case name?

Comment (valaparthvi) [reply cfme/tests/configure/test_visual_infra.py]: I think the test name reflects the case this test is testing, I'd like to keep it as is.

File: cfme/tests/configure/test_visual_infra.py [hunk 3]
```diff
@@ -365,8 +375,45 @@ def test_template_noquads(appliance, set_template_quad):
...
+        testSteps:
+            1. See if save button is enabled.
+            2. Save and check if the value was updated.
+        expectedResults:
+            1. Save button is enabled.
+            2. Value is changed successfully.
+    """
+    view = navigate_to(appliance.user.my_settings, "Visual")
+    visual = view.tabs.visual
+    available_options = [_.text for _ in visual.grid_tile_icons.long_text.all_options]
```
[the remark this answers is not in the document]
Comment (valaparthvi) [reply cfme/tests/configure/test_visual_infra.py]: Oh yes, didn't think of that! Good catch :+1:

**Pull request merged**